api/src/services/webhooks.ts

```diff
@@ -1,45 +1,56 @@
 import { AbstractServiceOptions, Item, PrimaryKey, Webhook, MutationOptions } from '../types';
 import { register } from '../webhooks';
 import { ItemsService } from './items';
+import { getMessenger, Messenger } from '../messenger';
 
 export class WebhooksService extends ItemsService<Webhook> {
+	messenger: Messenger;
+
 	constructor(options: AbstractServiceOptions) {
 		super('directus_webhooks', options);
+
+		this.messenger = getMessenger();
 	}
 
 	async createOne(data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
 		const result = await super.createOne(data, opts);
 		await register();
+		this.messenger.publish('webhooks', { type: 'reload' });
```

Comment:
Wondering if we can be a little smarter with the types here. Technically speaking, the channel and payload can both be any string, but it would be neat if we combine the two in TS where it knows what payload to expect based on the first parameter